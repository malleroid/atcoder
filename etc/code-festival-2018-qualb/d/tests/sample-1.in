1 3 100
1 30
3 20
9 50
