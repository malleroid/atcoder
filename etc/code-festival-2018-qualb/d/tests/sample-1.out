3.6000000000
